kythe/extractors/bazel/extract.sh

```diff
@@ -66,6 +66,13 @@ fi
 # care if some targets fail, but bazel will return a failure code if any
 # targets fail.
 /kythe/bazelisk --bazelrc=/kythe/bazelrc "$@" || true
+retval=$?
```

Comment:
Have you tested this? I'm pretty sure this will always be zero due to the `|| true` unless pipefail is set.  You probably want: `${PIPESTATUS[0]}` to get the bazelisk status regardless of the rest of the pipeline or shell options.

Reply:
Actually, that doesn't work either because it's not a pipeline, it's an `||`.  I think you'll need:
```
if !/kythe/bazelisk ...; then
  ...
fi
```

Reply:
Good catch, thanks. I changed from `|| true` to `|| retval=$?` and tested it this time.